.travis.yml

```diff
@@ -39,11 +39,17 @@ before_script:
 
 script:
   - make quality
-  - make test
-  # Check if emulator has finished booting
-  - android-wait-for-emulator
-  - adb shell input keyevent 82 &
-  - travis_wait make e2e
+
+# Disable test cases for some time due to dependency on updating of roboelectric library which we
+# shall cover in story LEARNER-5469
+# Simple assemble the prod build to avoid running of 'gradle build connectedCheck' command which
+# runs the test cases.
+  - ./gradlew assembleProdDebug
```

Comment:
Instead of this we should have:
```java
 - make quality
```
Also, the explanation above could be bundled with a `TODO` or `FIXME` for better explanation.

p.s. its Robolectric 😉 
p.p.s. took me some time too, to recognize that they didn't use the initial `e` of electric in the name 😆 

Keeping the above points in mind:

1. commit msg needs to update as well.
2. LEARNER-5469 should also be updated as to what happened here with proper link-backs.